### PR TITLE
fix(sheet): restore code in setStyleCommand

### PR DESCRIPTION
**Pull request by siam-ese**
Incorrectly change the original code. to fix it.  Close [#685](https://github.com/dream-num/univer-pro/issues/685#event-12775832720)

### CODE REVIEW

**Comment by wzhudev** [comment]
Please add a unit test.

**Comment by siam-ese** [comment]
> Please add a unit test.

The test has been added.

**Comment by siam-ese** [comment]
> > Please add a unit test.
> 
> The test has been added. Please review it again.

File: packages/sheets/src/commands/commands/__tests__/set-style.command.spec.ts [hunk 1]
```diff
@@ -14,7 +14,8 @@
  * limitations under the License.
  */
 
-import type { ITextDecoration, ITextRotation, Univer } from '@univerjs/core';
+import type { IColorStyle, ITextDecoration, ITextRotation,
```
[the remark this answers is not in the document]
Comment (siam-ese) [reply packages/sheets/src/commands/commands/__tests__/set-style.command.spec.ts:17]: image

[the remark this answers is not in the document]
Comment (siam-ese) [reply packages/sheets/src/commands/commands/__tests__/set-style.command.spec.ts:17]: No new line, it's changing at the same line.

File: packages/sheets/src/commands/commands/__tests__/set-style.command.spec.ts [hunk 2]
```diff
@@ -81,6 +82,52 @@ describe("Test commands used for updating cells' styles", () => {
 
     afterEach(() => univer.dispose());
 
+    describe('set style', () => {
```
Comment (wzhudev) [inline packages/sheets/src/commands/commands/__tests__/set-style.command.spec.ts:85]: No need to wrap the case in a `describe`.

**Review by wzhudev — APPROVED** [review]

**Pull request merged**